todo-app/src/components/addingContainer.js

```diff
@@ -0,0 +1,43 @@
+
+import React, { Component } from "react";
+import FaPlus from "react-icons/lib/fa/plus"
+
+
+export default class AddingContainer extends Component {
+    constructor(props) {
```

Comment:
As we discussed in the week 2 class, it's not necessary to use a constructor unless you really want to! Up to you.